content/en/docs/reference/access-authn-authz/node.md

```diff
@@ -34,8 +34,8 @@ Write operations:
 
 Auth-related operations:
 
-* read/write access to the certificationsigningrequests API for TLS bootstrapping
-* the ability to create tokenreviews and subjectaccessreviews for delegated authentication/authorization checks
+* read/write access to the certification signing requests API for TLS bootstrapping
+* the ability to create tokenreviews and subject access reviews for delegated authentication/authorization checks
```

Comment:
I guess that

- `certificationsigningrequests` is a typo of [`CertificateSigningRequest`](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/)
- `tokenreviews` stands for `tokenreviews.authentication.k8s.io`, `authentication.k8s.io/v1/tokenreviews` or `authentication.k8s.io/v1beta1/tokenreviews`
- `subjectaccessreviews` stands for `authorization.k8s.io/v1/subjectaccessreviews` or `authorization.k8s.io/v1beta1/subjectaccessreviews`

and `certification signing requests` & `subject access reviews` are not appropriate expressions.

So I suggest:
```suggestion
* read/write access to the [`CertificateSigningRequest`](/docs/reference/access-authn-authz/certificate-signing-requests/) API for TLS bootstrapping
* the ability to create tokenreviews and subjectaccessreviews for delegated authentication/authorization checks
```

Reply:
Makes sense! thanks!